elements/rh-card/demo/rh-card.html

```diff
@@ -0,0 +1,128 @@
+<link rel="stylesheet" href="demo.css">
+<script type="module" src="rh-card.js"></script>
+<div id="demo-container">
+  <div>
+    <p>Card / Basic</p>
+    <p>Component state: <b>Default</b></p>
+    <rh-card>
+      <h2>Headline, sm</h2>
+      <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est egestas, a
+          sollicitudin mauris tincidunt. Pellentesque vel dapibus risus. Nullam aliquam felis orci, eget cursus mi
+          lacinia quis. Vivamus at felis sem.
+      </p>
+      <rh-cta priority="primary" slot="footer">
+        <a href="#">Call to action</a>
+      </rh-cta>
+    </rh-card>
+  </div>
+  <div>
+    <p>Component state: <b>Default, slotted header</b></p>
+    <rh-card>
+      <h2 slot="header">Headline, sm</h2>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend elit sed est egestas, a
+        sollicitudin mauris tincidunt. Pellentesque vel dapibus risus. Nullam aliquam felis orci, eget cursus mi
+        lacinia quis. Vivamus at felis sem.
+      </p>
+      <rh-cta priority="primary" slot="footer">
+        <a href="#">Call to action</a>
+      </rh-cta>
+    </rh-card>
+  </div>
+  <div>
+    <p>Card / Basic - Header</p>
+    <p>Component state: <b>Default</b></p>
+    <rh-card bar>
```

Comment:
bar?

Reply:
Those were leftover demos from when they were variants, I've updated this